package.json

```diff
@@ -33,7 +34,7 @@
     "tslint": "tslint --config tslint.json --format stylish --project .",
     "format": "prettier --write \"*.{css,js,json,md,scss,ts,tsx}\" \"./**/*.{css,js,json,md,scss,ts,tsx}\"",
     "transpile": "tsc",
-    "clean-transpile": "rimraf ts/**/*.js ts/*.js",
+    "clean-transpile": "rimraf ts/**/*.js ts/*.js ts/protobuf/*.d.ts",
```

Comment:
Seems like this shouldn't kill things under `ts/protobuf` because those aren't generated by `yarn transpile`. I would expect `clean-transpile` to be reversible with `transpile`. So perhaps we need a `clean-protobuf`?